### ISSUE TITLE
Input Form

### ISSUE THREAD

**Issue by Mogax** [Mogax/a11y-project#36]
**Wich page**
Login & Register

**What is wrong**
Input password do not prevent users from pasting.
Field are automatically in error style.
Button register is activate also when password and confirm password are differents
Tab Index are not correctly setup
Missing Input Validation

**Additional info**
